src/config.c

```diff
@@ -230,6 +230,7 @@ bool Config_ReadFromJSON(const char *cfg_data)
     READ_BOOL(enable_ps_uzi_sfx, false);
     READ_BOOL(enable_jump_twists, true);
     READ_BOOL(enabled_inverted_look, false);
+    READ_INTEGER(camera_speed, 5);
```

Comment:
Is it worth clamping the value here (similar to hitpoints, FoV etc) in case somebody edits the config file manually with a bad value?

Reply:
Yes